sympy/external/gmpy.py

```diff
@@ -37,6 +37,12 @@
 
     # gcd from gmpy or math
     'gcd',
+
+    # invert from gmpy or pow
+    # If inverse element does not exist,
+    # gmpy raises `ZeroDivisionError` and
+    # pow raises `ValueError`.
+    'invert',
```

Comment:
It would be better to wrap `pow` with something that raises the same error as `gmpy` (or the other way around). The intention of this module is that anything that imports from here should not need to care about whether the `invert` function comes from gmpy or not because its behaviour should be the same either way.

Reply:
For performance it would be better to wrap `pow` rather than `invert` because a `lambda` function is already being used and the overhead of wrapping would more noticeable for `gmpy.invert`. I suggest:
```python
def invert(x, m):
   """Some docstring"""
    try:
        return pow(x, -1, m)
    except ValueError:
        raise ZeroDivisionError("invert() no inverse exists")
```